io/pci/EEH.py

```diff
@@ -105,11 +106,19 @@ def test_eeh_basic_pe(self):
         enter_loop = True
         num_of_miss = 0
         num_of_hit = 0
-        while num_of_hit <= self.max_freeze:
+        while num_of_hit < self.max_freeze:
```

Comment:
so we are injecting less than the max_freeze, we should still keep eeh more than max_freeze as last scenario @Naresh-ibm any suggestions here ?

Reply:
@abdhaleegit  yes, driver will recover till max_freez_value and goes down on next eeh trigger.
means if we give max_free value as 5 then driver has to recover till 5th time and it will fails to recover on 6th time. hence we have to validate the driver recover till max time value and we have trigger the injection for max+1th time. so the condition for validation to be like below
`while num_of_hit <= self.max_freeze:`

Reply:
Sure. @abdhaleegit and @Naresh-ibm you are correct. I would like to make changes as follows
Read max freeze value from OS if no value is not passed in yaml.
Please let me know your suggestion on this.